native/database/db.go

```diff
@@ -0,0 +1,132 @@
+package database
```

Comment:
native should be a package to connect to Native chain. The relayer DB should be in a separate package, so let's move it to the root.